src/Symfony/Component/Console/Command/TraceableCommand.php

```diff
@@ -0,0 +1,356 @@
+     */
+    public function setProcessTitle(string $title): static
+    {
+        $this->command->setProcessTitle($title);
+
+        return parent::setProcessTitle($title);
+    }
+
+    public function setHelp(string $help): static
+    {
+        $this->command->setHelp($help);
+
+        return $this;
+    }
+
+    public function getHelp(): string
+    {
+        return $this->command->getHelp();
+    }
+
+    public function getProcessedHelp(): string
+    {
+        return $this->command->getProcessedHelp();
+    }
+
+    public function getSynopsis(bool $short = false): string
+    {
+        return $this->command->getSynopsis($short);
+    }
+
+    public function addUsage(string $usage): static
+    {
+        $this->command->addUsage($usage);
+
+        return $this;
+    }
+
+    public function getUsages(): array
+    {
+        return $this->command->getUsages();
+    }
+
+    public function getHelper(string $name): HelperInterface
+    {
+        return $this->command->getHelper($name);
+    }
+
+    public function run(InputInterface $input, OutputInterface $output): int
+    {
+        $this->input = $input;
+        $this->output = $output;
+        $this->arguments = $input->getArguments();
+        $this->options = $input->getOptions();
+        $event = $this->stopwatch->start($this->getName(), 'command');
+
+        try {
+            $this->exitCode = parent::run($input, $output);
+        } finally {
+            $event->stop();
+
+            if ($output instanceof ConsoleOutputInterface && $output->isDebug()) {
+                $output->getErrorOutput()->writeln((string) $event);
+            }
+
+            $this->duration = $event->getDuration().' ms';
+            $this->maxMemoryUsage = ($event->getMemory() >> 20).' MiB';
+
+            if ($this->isInteractive) {
+                $this->extractInteractiveInputs($input->getArguments(), $input->getOptions());
+            }
+        }
+
+        return $this->exitCode;
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output): void
+    {
+        $event = $this->stopwatch->start($this->getName().'.init', 'command');
+
+        $this->command->initialize($input, $output);
+
+        $event->stop();
+    }
+
+    protected function interact(InputInterface $input, OutputInterface $output): void
+    {
+        if (!$this->isInteractive = Command::class !== (new \ReflectionMethod($this->command, 'interact'))->getDeclaringClass()->getName()) {
+            return;
+        }
+
+        $event = $this->stopwatch->start($this->getName().'.interact', 'command');
+
+        $this->command->interact($input, $output);
+
+        $event->stop();
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $event = $this->stopwatch->start($this->getName().'.execute', 'command');
+
+        $exitCode = $this->command->execute($input, $output);
```

Comment:
I noticed that this call does not work if the method is defined in a trait.
https://3v4l.org/AFJnQ
This is the case for `DoctrineFixturesBundle` with the [`CommandCompatibility` trait]( https://github.com/doctrine/DoctrineFixturesBundle/blob/3.5.1/Command/CommandCompatibility.php)

I don't think it's worth opening an issue.